dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/manually-set-completed/manually-set-completed.event-handler.test.ts

```diff
@@ -1,14 +1,45 @@
-import { UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import { EntityManager } from 'typeorm';
+import {
+  DbRequestSource,
+  UTILISATION_REPORT_RECONCILIATION_STATUS,
+  UtilisationReportEntity,
+  UtilisationReportEntityMockBuilder,
+  getDbAuditUpdatedByUserId,
+} from '@ukef/dtfs2-common';
 import { handleUtilisationReportManuallySetCompletedEvent } from './manually-set-completed.event-handler';
-import { NotImplementedError } from '../../../../../errors';
 
 describe('handleUtilisationReportManuallySetCompletedEvent', () => {
-  // TODO FN-1862 - update tests when functionality implemented
-  it('throws a NotImplementedError', () => {
+  const requestSource: DbRequestSource = {
+    platform: 'TFM',
+    userId: 'abc123',
+  };
+  const updatedByUserId = getDbAuditUpdatedByUserId(requestSource);
+
+  const mockSave = jest.fn();
+
+  const mockRepository = {
+    save: mockSave,
+  };
+
+  it('calls the correct repo methods and updates the report', async () => {
```

Comment:
Again same comment re correct, but also here we should also spell out what we mean by 'updates'.